docs/v0.7/en/user-guide/operations/configuration.md

```diff
@@ -338,18 +338,19 @@ dir = "/tmp/greptimedb/logs"
 level = "info"
 enable_otlp_tracing = false
 otlp_endpoint = "localhost:4317"
-tracing_sample_ratio = 1.0
 append_stdout = true
+[logging.tracing_sample_ratio]
+default_ratio = 0.0
```

Comment:
is it recommended to use 0 as the default ratio?

Reply:
Same as above

Reply:
the config in configuration has changed